src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss

```diff
@@ -27,4 +27,11 @@ $button-spacing: rem(12px);
       @include focus-ring($border-width: rem(0));
     }
   }
+
+  &.destructive {
+    a,
+    button {
+      color: var(--p-text-critical);
```

Comment:
Did you confirm with Polaris this was the expected result?

Because [with Card](https://codesandbox.io/s/suspicious-yalow-jy91k?file=/App.js) they render a red button when a secondary footer action is destructive. I just wonder if we should be consistent with that.

Reply:
I think this is fine in the context I have but we need hover / focus states for this as well I believe. Also, because these actions were pretty hackily built (you're welcome 😄) you might need to include `!important`